compiler/src/main/scala/org/scalajs/nscplugin/ExplicitLocalJS.scala

```diff
@@ -212,7 +212,8 @@ abstract class ExplicitLocalJS[G <: Global with Singleton](val global: G)
           for (decl <- decls) {
             decl match {
               case ClassDef(_, _, _, impl)
-                  if decl.symbol.isModuleClass && isInnerJSClassOrObject(decl.symbol) =>
+                  if decl.symbol.isModuleClass && isInnerJSClassOrObject(
```

Comment:
I think this is a manual change to but wrapping after the `&&` is nicer to keep the function `isInner...` together. In general this is what is nice about scalafmt; if the copy wraps and it is not desired, you can usually adjust the wrap and get something nicer.